tools/ja-rule/OpenLightingDevice.cpp

```diff
@@ -140,10 +131,11 @@ bool OpenLightingDevice::Init() {
     return false;
   }
 
-  r = libusb_claim_interface(m_handle, 0);
+  r = libusb_claim_interface(m_handle, INTERFACE_OFFSET);
```

Comment:
Well on the device side it's an offset. You specify the number interfaces each function driver uses and it computes the value for you.